tests/test_meshtastic_utils_node_name_refresh.py

```
@@ -123,6 +123,56 @@ def test_refresh_node_name_tables_handles_timeout_then_retries(
+    assert len(unavailable_calls) == 1
+
+
+def test_refresh_node_name_tables_uses_reconnecting_unavailable_message(
+    reset_meshtastic_globals,
+) -> None:
+    """When reconnecting, refresh logs reconnect-specific unavailability reason."""
+    _ = reset_meshtastic_globals
+    event = _OnePassEvent()
+    with (
+        patch.object(mu, "meshtastic_client", None),
+        patch.object(mu, "reconnecting", True),
```

Ruff (0.15.9)

[warning] Boolean positional value in function call

(FBT003)